helm_lib/templates/_envs_for_proxy.tpl

```diff
@@ -1,6 +1,8 @@
 {{- /* Usage: {{ include "helm_lib_envs_for_proxy" . }} */ -}}
+{{- /* Add HTTP_PROXY, HTTPS_PROXY and NO_PROXY environment variables for container */ -}}
+{{- /* depends on [proxy settings](https://deckhouse.io/documentation/v1/deckhouse-configure-global.html#parameters-modules-proxy) */ -}}
 {{- define "helm_lib_envs_for_proxy" }}
-  {{- $context := . -}}
+  {{- $context := . -}} {{- /* Dot object (.) with .Values, .Chart, etc */ -}}
```

Comment:
- I'd be more precise on template context rather than 'dot object'
- I'd rather not write "etc", if we have fixed number of first-level fields in the context

```suggestion
  {{- $context := . -}} {{- /* Template context with .Values, .Chart, etc */ -}}
```